lib/views/changed-files-count-view.js

```diff
@@ -24,7 +30,7 @@ export default class ChangedFilesCountView extends React.Component {
       <button
         ref="changedFiles"
         className="github-ChangedFilesCount inline-block"
-        onClick={this.props.didClick}>
+        onClick={this.handleClick.bind(this)}>
```

Comment:
@smashwilson : would it be better to use `autobind` here for consistency's sake even though there's only one method to bind and no constructor in this component?

Reply:
@smashwilson thanks for the extra context!  I was just being lazy. :-p . I'll fix it.